test/modules/aws-account-integration.ts

```diff
@@ -45,6 +45,10 @@ describe('AwsAccount Integration Testing', () => {
 
   it('does absolutely nothing when you sync this', sync());
```

Comment:
Should these be moved from the direct function call mechanism to the `query`-based version?

Reply:
this is a shorthand for that defined in `helpers`

Reply:
Oh, we changed the helper instead of deleting it? Hmm... We did that before because invoking the iasql.ts file directly was a little awkward, but now if we switch to 100% `query`-based testing the tests themselves can educate readers on how to use IaSQL.

We could make that a follow-up task?

Reply:
> Oh, we changed the helper instead of deleting it? 

Yes

> Hmm... We did that before because invoking the iasql.ts file directly was a little awkward, but now if we switch to 100% `query`-based testing the tests themselves can educate readers on how to use IaSQL. We could make that a follow-up task?
 
What you are describing is already the case. The reason none of the quickstart or documentation reference has been added yet is because of https://github.com/iasql/iasql-engine/issues/539, but that will be fairly trivial to do after https://github.com/iasql/iasql-engine/pull/542

Reply:
I meant having the test suite itself also act as documentation by always using `query(...` instead of the existing helpers.

Reply:
oh, sure I see your point. hmm eventually it will be all over the documentation so it doesn't seem super necessary especially since they can look at the declaration in the `helper`? the `install`-ing of multiple modules doesn't look great to put all over the place with the query replacement https://github.com/iasql/iasql-engine/blob/main/test/helpers.ts#L40 ? but otherwise I'm game for it

Reply:
created https://github.com/iasql/iasql-engine/issues/544